mkdocs.navigation.yml

```diff
@@ -188,6 +188,7 @@ nav:
               - private-networks/tutorials/permissioning/index.md
               - Get started with onchain permissioning: private-networks/tutorials/permissioning/onchain.md
               - Upgrade permissioning contracts: private-networks/tutorials/permissioning/upgrade-contracts.md
+          - Deploy Besu with Sirato explorer: private-networks/tutorials/sirato-explorer.md
```

Comment:
Avoid this as a seperate page and keep the context of the quickstart. Additionally add a line or two in the monitoring page here https://besu.hyperledger.org/en/latest/private-networks/how-to/monitor/?h=monitoring